migrations/00003_app_grants.sql

```diff
@@ -0,0 +1,9 @@
+-- +goose Up
+-- +goose StatementBegin
+GRANT ALL PRIVILEGES ON SCHEMA shortener TO postgres;
```

Comment:
[CHANGE REQUEST] It should not do it in the migrations because of security reasons.